noodles-editor/src/noodles/components/menu.tsx

```diff
@@ -679,6 +683,36 @@ export function NoodlesMenubar({
             </Menubar.Content>
           </Menubar.Portal>
         </Menubar.Menu>
+
+        <Menubar.Menu>
+          <Menubar.Trigger className={s.menubarTrigger}>View</Menubar.Trigger>
+          <Menubar.Portal>
+            <Menubar.Content
+              className={s.menubarContent}
+              align="start"
+              sideOffset={5}
+              alignOffset={-3}
+            >
+              <Menubar.Item className={s.menubarItem}>Hide Nodes</Menubar.Item>
+              <Menubar.Item className={s.menubarItem}>Hide TheatreJS</Menubar.Item>
+              <Menubar.Separator className={s.menubarSeparator} />
+              <Menubar.Item className={s.menubarItem}>Toggle Fullscreen</Menubar.Item>
+            </Menubar.Content>
+          </Menubar.Portal>
+        </Menubar.Menu>
+
+        {/* Chat button on the right side */}
+        {setShowChatPanel && (
+          <div className={s.menubarRightSlot}>
```

Comment:
Can this layout be done with radix?

Reply:
Yeah I mean it's not winning any prizes but it looks like this
<img width="3024" height="100" alt="Screenshot 2025-10-18 at 11 23 04 AM" src="https://github.com/user-attachments/assets/3b193787-5c81-4cf2-aeb9-bc9a608dba44" />